README.md

```diff
@@ -48,41 +48,67 @@ Questionable Conditions that the application will verify with User via A Dialog
 - Output File does not end with .pdf, verify with user that's really what they want
 - Output File Name Matches a File in the Output Directory
 
+
+---
+
+## Beofre Installation
+
+### 1. Download Python
+Users are required to have Python3 installed in their machine before using the application.
```

Comment:
Python should actually not be needed cause it was compiled into the Mac App bundle.... what happens if you removed python and ran the app again? Thanks!

Reply:
```
python3 iDensify.py
```
This line of command failed to compile when I didn't download python3. At least I do not know what are the alternatives to that matter.

Reply:
Ahhh yes... so if you want to run the script with Python, then you have to install Python. I agree I should add that as a step before running it as a Python script even though I assumed that anyone that would want to run the script with Python is an existing Python user or developer. Otherwise, they should just run the binary. Thanks!

Reply:
Lol no problem, I was thinking that too lol but I'm guessing not all "developers" are Python users, which there will be some blanks to fill lololol